integration/testcmd/mountkexec/uinit/mountkexec.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+
+	"github.com/u-root/u-root/pkg/sh"
+	webboot "github.com/u-root/webboot/pkg/mountkexec"
+	"golang.org/x/sys/unix"
+)
+
+// Mount a vfat volume and kexec the kernel within.
+func main() {
+
+	if err := webboot.MountISO("/dev/sda1", ""); err != nil {
+		log.Printf(err.Error())
+	}
+
+	if err := webboot.MountISO("/dev/sd", "mountDir"); err != nil {
+		log.Printf(err.Error())
+	}
+
+	if err := webboot.MountISO("/dev/sda1", "mountDir"); err != nil {
+		log.Fatal(err)
+	}
+
+	if err := sh.Run("ls", "mountDir/"); err != nil {
+		log.Fatal(err)
```

Comment:
This is not enough. I recommend you change it:
if o, err := exec.Command(""ls", "mountDir").CombinedOutput(); err != nil {
log.Fatalf("ls failed: %s, error %v", string(o), err)
}

Reply:
Done